plugins/GSdx/GSRendererHW.cpp

```diff
@@ -186,6 +186,42 @@ void GSRendererHW::SetGameCRC(uint32 crc, int options)
 	GSRenderer::SetGameCRC(crc, options);
 
 	m_hacks.SetGameCRC(m_game);
+
+	// Code for Automatic Mipmapping. Relies on game CRCs.
+	CRC::Game game_mipmap; game_mipmap = CRC::Lookup(crc);
```

Comment:
Actually, do we even need a local variable here? I think just using ``switch (CRC::Lookup(crc).title) {`` will be enough. Sorry for not mentioning about it earlier! It just struck my mind right now and I suck at properly reviewing things.